internal/commands/app/create_inputs.go

```diff
@@ -83,34 +81,39 @@ func (i *createInputs) resolveDirectory(wd string) (string, error) {
 	if !fi.Mode().IsDir() {
 		return fullPath, nil
 	}
-	_, appOK, appErr := local.FindApp(fullPath)
-	if appErr != nil {
-		return "", appErr
+	_, appOK, err := local.FindApp(fullPath)
+	if err != nil {
+		return "", err
 	}
 	if appOK {
-		return "", fmt.Errorf("%s is inside or is a Realm app directory", dir)
+		return "", errProjectExists{details: fmt.Sprintf("%s is inside or is a Realm app directory", fullPath)}
```

Comment:
if we make the switch to `path string` for this error struct, this can be just `errProjectExists{"./"+i.Directory}`